validator/src/server/cli.rs

```diff
@@ -44,11 +44,11 @@ pub fn wrap_in_pydict(py: Python, matches: &ArgMatches) -> PyResult<PyDict> {
         py,
         "minimum_peer_connectivity",
         matches
-            .value_of("minimum-peer-connectivity")
+            .value_of("minimum_peer_connectivity")
```

Comment:
This is the CLI parameter for rust version, it then is mapping to python dictionary `minimum_peer_connectivity`. Can you confirm which version of Sawtooth you're currently on?

Reply:
Response from my colleague who implemented the fix in our form:

> Right, the issue is that the value for the parameter is stored under the key "minimum_peer_connectivity" (https://github.com/hyperledger/sawtooth-core/blob/7651cb5552800dbac98559b46339a84eae472ef7/validator/src/server/cli.rs#L187) but then the value is looked up by the key "minimum-peer-connectivity" (https://github.com/hyperledger/sawtooth-core/blob/7651cb5552800dbac98559b46339a84eae472ef7/validator/src/server/cli.rs#L47) which doesn't exist, so it always falls back to the default value of 3 (https://github.com/hyperledger/sawtooth-core/blob/7651cb5552800dbac98559b46339a84eae472ef7/validator/sawtooth_validator/config/validator.py#L201).

Reply:
Thanks for the explanation. I see inconsistency there. We may need to back-port and release a patch if required.